### PR TITLE
Moving thing in my-app to root folder.

### PR DESCRIPTION
**Pull request by jhoover4**
Moving things from `my-app` to root of project.

We're doing this because a proper app setup for a single react app in a repo should have `src` as a top level folder and `package.json` in the root.

### CODE REVIEW

File: my-app/README.md
Comment (ProsperousHeart) [inline my-app/README.md:1]: Since there were no changes in this README (from **create-react-app**), the deletion is fine. In the future, please confirm with the team if you are deleting files you did not create. @jhoover4

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Unable to approve, since running this branch causes an error:
image

you are missing the entire public folder

File: my-app/README.md
Comment (jhoover4) [reply my-app/README.md:1]: Thats way too much overhead. PRs are so that maintainers can check if deleted files are okay or not.

Comment (jhoover4) [reply my-app/README.md:1]: Also that error appears on the `v1-mvp` branch so its an existing error. I can fix it here, but my changes didn't introduce it. All this did was move what was already there

Comment (ProsperousHeart) [reply my-app/README.md:1]: It works when you change directory into the **my-app** folder and run **npm start** - this is one of the reasons why they needed to be moved. The other being that GitHub Pages also looks at the source folder.

Comment (jhoover4) [reply my-app/README.md:1]: Looks like the issue was that we had a `.gitignore` rule that was ignoring all `public` folders. That shouldn't be there, I'm removing it.

File: .gitignore
Comment (jhoover4) [inline .gitignore:61]: This is introducing issues where public folders aren't commited correctly

Comment (ProsperousHeart) [reply .gitignore:61]: Thank you for bringing this to our attention. I'll go ahead and take care of this since I was unaware that folder was in `.gitignore` (I assume it's from the prior code pre 2023).

**Comment by ProsperousHeart** [comment]
To avoid the back and forth and to ensure a clean branch for volunteers moving forward (plus provide other updates needed) we will now be closing this PR to #53. Once it is merged, one of the core admin team members will comment on #49 so you can pull updates from the **v1-MVP** branch and push back up to #49.

Thank you for your patience and insight.

File: my-app/README.md
Comment (jhoover4) [reply my-app/README.md:1]: @ProsperousHeart nope it never worked. It works for you because you did this yourself, but `public` never got pushed for everyone else.

Comment (ProsperousHeart) [reply my-app/README.md:1]: I am aware, thank you.

**Pull request closed without merging**
